### PR TITLE
Create Prompt.md

### PR DESCRIPTION
**Pull request by slinaresGX**
New article added.

### CODE REVIEW

File: saia-docs/RAG/Prompt.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+The default value is:
+```
+Given the following conversation and a follow-up question, rephrase the follow-up question to be a standalone question.
+
+Chat History:
+{chat_history}
+Follow-Up Input: {question}
+Standalone question:
+```
+## LLM Settings
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:58]: The photo shows Lml settings and not LLM Settings.

File: saia-docs/RAG/Prompt.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+“bedrock”.
+
+* ### apiKey
+
+API authentication key provided to access the language model service.

For example, if the Provider parameter is "azureopenai", you must 
+specify "apiKey" with the authentication key.
+
+* ### modelName
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:74]: The photo shows Model Name and not modelName.

File: saia-docs/RAG/Prompt.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+It specifies the name of the language model used.

For example, if the Provider parameter has the value "openai", modelName takes the value 
+'gpt-3.5-turbo'.
+
+* ### temperature
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:79]: The photo shows Temperature and not temperature. Maybe it has a reason, but just letting you know about it.

File: saia-docs/RAG/Prompt.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+It controls the diversity of generated answers. It is expressed as decimal numeric values, where a value of 1 indicates that all possible 
+options are considered. 


+
+This parameter allows regulating the diversity of the generated answers, determining the extent of the options considered during the text 
+generation process.
+
+* ### stream
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:97]: The picture shows Stream and not stream.

File: saia-docs/RAG/Prompt.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+It enables or disables the transmission of results in a continuous or non-continuous manner during the process. It is configured using 
+Boolean values, where 'true' enables the transmission and 'false' disables it, so the end result is obtained directly.
+
+* ### verbose
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:102]: The picture shows Verbose and not verbose.

File: saia-docs/RAG/Prompt.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+It enables or disables the level of detail in the information generated by the model during the process. It can be configured with Boolean 
+values, where 'true' enables the detailed output and 'false' disables it. 
+
+* ### cache
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:107]: The picture shows Cache and not cache.

File: saia-docs/RAG/Prompt.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+It establishes the text generation process by regulating the appearance of certain terms in the answers. It is configured with decimal 
+numeric values from -2.0 to 2.0.   More information: https://platform.openai.com/docs/api-reference/chat/create#chat-create-frequency_penalty
+
+* ### endpoint
+
+It configures the URL pointing to the specific server or service where the language models are hosted. 
+
+The endpoint configuration varies depending on the service provider used.
+
+For example, for 'openai', the URL must be precise and contain the specific engine version, for example: https://api.openai.com/v1/chat/completions
```
Comment (Valentina-Deambrosio10) [inline saia-docs/RAG/Prompt.md:128]: I would add a period after completions. Can you add it, please? Thanks.

**Review by Valentina-Deambrosio10 — COMMENTED** [review]
Several comments to review.

File: saia-docs/RAG/Prompt.md (hunk 1 above)
Comment (slinaresGX) [reply saia-docs/RAG/Prompt.md]: done!

File: saia-docs/RAG/Prompt.md (hunk 2 above)
Comment (slinaresGX) [reply saia-docs/RAG/Prompt.md]: done!

File: saia-docs/RAG/Prompt.md (hunk 3 above)
Comment (slinaresGX) [reply saia-docs/RAG/Prompt.md]: done!

File: saia-docs/RAG/Prompt.md (hunk 4 above)
Comment (slinaresGX) [reply saia-docs/RAG/Prompt.md]: done!

File: saia-docs/RAG/Prompt.md (hunk 5 above)
Comment (slinaresGX) [reply saia-docs/RAG/Prompt.md]: done!

**Review by Valentina-Deambrosio10 — APPROVED** [review]
Approved.

**Pull request merged**